co-simulation/bundle/src/assembly/resources/etc/runtime.json

```diff
@@ -121,6 +121,20 @@
             ],
             "javaClasspathEntries": []
         },
+        {
+            "id": "carma-cloud",
+            "classname": "org.eclipse.mosaic.fed.carmacloud.ambassador.CarmaCloudMessageAmbassador",
+            "configuration": "carma-cloud_config.json",
+            "priority": 50,
+            "host": "local",
+            "port": 0,
+            "deploy": false,
+            "start": false,
+            "subscriptions": [
+                "V2xMessageReception"
```

Comment:
@chengyuan0124 Does the carma-cloud ambassador do anything with V2XMessageReception interactions?

Reply:
It doesn't, but the rule for subscriptions seems to require at least one entry in this section. I'll double-check to see if we can use an empty string here; otherwise, we might need to leave it as is.